swagger/reflect.go

```diff
@@ -220,7 +271,10 @@ func defineObject(v interface{}) Object {
 			}
 		}
 
-		p := inspect(field.Type, field.Tag.Get("json"), field.Tag.Get("format"))
+		p := inspect(field.Type, field.Tag.Get("json"), field.Tag.Get("format"),
```

Comment:
Is it worth just passing `(field.Type, field.Tag)` here? We can still pass `(t, "")` above, since `reflect.StructTag` is just a `string` underneath.